specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2023-08-01-preview/inference.json

```diff
@@ -0,0 +1,1373 @@
+                  "properties": {
+                    "object": {
+                      "type": "string"
+                    },
+                    "model": {
+                      "type": "string"
+                    },
+                    "data": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "index": {
+                            "type": "integer"
+                          },
+                          "object": {
+                            "type": "string"
+                          },
+                          "embedding": {
+                            "type": "array",
+                            "items": {
+                              "type": "number"
+                            }
+                          }
+                        },
+                        "required": [
+                          "index",
+                          "object",
+                          "embedding"
+                        ]
+                      }
+                    },
+                    "usage": {
+                      "type": "object",
+                      "properties": {
+                        "prompt_tokens": {
+                          "type": "integer"
+                        },
+                        "total_tokens": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "prompt_tokens",
+                        "total_tokens"
+                      ]
+                    }
+                  },
+                  "required": [
+                    "object",
+                    "model",
+                    "data",
+                    "usage"
+                  ]
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/deployments/{deployment-id}/chat/completions": {
+      "post": {
+        "summary": "Creates a completion for the chat message",
+        "operationId": "ChatCompletions_Create",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "deployment-id",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "description": "Deployment id of the model which was deployed."
+            }
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "example": "2023-08-01-preview",
+              "description": "api version"
+            }
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "messages": {
```

Comment:
are functions gone? They were included in the 2023-07-01-preview API versions....

Reply:
definitely still supported and agreed that functions changes need to be restored.
RAI content filter results appear to be in there so it's likely just 2023-07-01-preview deltas (which should just be functions) that weren't ported -- though re-checking a full diff is still a good idea.

Reply:
Thanks for catching it! The functions were not there when I reviewed the PR with the board, that is why they are missing. Let me cherry pick the functions related changes to Aug.